packages/slate/src/interfaces/operation.ts

```diff
@@ -228,9 +228,18 @@ export const Operation = {
           return op
         }
 
-        // We need to get the original path here, but sometimes the `newPath`
-        // is a younger sibling of (or ends before) the original, and this
-        // accounts for it.
+        // If the move happens completely within a single parent the path and
```

Comment:
This helped more than I expected, it wasn't immediately obvious that `Path.isSibling` was within a single parent until I took a close look at the code 😂